### DISCUSSION TITLE
Comparto mis practicas CSS y JS, un dolor de cabeza...

### DISCUSSION THREAD

**Discussion by leandrho** [Launch-X-Latam/MisionFrontEnd#383] [Q&A]
Vaccination
https://sparkly-biscochitos-5e353a.netlify.app/
Pokedex
https://splendorous-puppy-8e1496.netlify.app/

Puede que en distintas pantallas haya un par de bugs y el codigo sea espagueti, pero hasta ahi me dio el tiempo.
Saludos

**Comment by floressdi** [comment]
Me encanto ¿Donde conseguiste las imagenes?

**Reply by leandrho** [reply]
No recuerdo la pagina, pero busque con google image y las edite un poco.
Te las paso en fullhd
Espero que te sirvan
Saludos


![f1](https://user-images.githubusercontent.com/37672074/201245887-bb5926a2-9448-4d0a-b9b2-b6950b6c6233.jpg)
![f2](https://user-images.githubusercontent.com/37672074/201245795-c5099bfb-99f9-45e5-9f58-22287c4faeb6.jpg)
![f3](https://user-images.githubusercontent.com/37672074/201245908-fe740f17-69f4-4303-abf5-f1e66a7f07d5.jpg)

**Reply by floressdi** [reply]
Muchas Gracias!!

**Comment by AnaMejiaR** [comment]
Perdona el código para realizar las páginas están en algún archivo?, o solo las van elaborando conforme lo que se ve en los en vivos del Sherpa, me podrías responder por favor, muchas gracias y disculpa la molestia.

**Reply by leandrho** [reply]
La practica css(vaccination) hay que hacerla desde cero, tanto el html como el css. Para la de js(pokedex) tambien hay que hacer el html, css y js desde cero, pero para esta tenes en github - mision frontend - 04-JS, un ejemplo de como hacer uso del api pokedex que lo podes usar. Espero te sirva la info. Saludos
